Ticket: #54690 API Inbound - CreateTagForPO - FG

vishnuvellankijkt opened the pull request:
Files:
api\inbound\handlers\CreateTagForPORequestHandler.p.p
api\inbound\CreateTagForPO.p

vishnuvellankijkt committed:
Ticket: #54690 API Inbound - CreateTagForPO - FG

Files:
api\inbound\handlers\CreateTagForPORequestHandler.p.p
api\inbound\CreateTagForPO.p

vishnuvellankijkt committed:
Ticket: #54690 API Inbound - CreateTagForPO - FG

brad-advantzware reviewed (changes requested):
"KELLOGS" should not be hard-coded.  If neither the po-ordl.cust-no or oe-ord.cust-no, then cust-no is "".
Need to test availability of oe-ord before using oe-ord.cust-no.
"cust-par" table is "cust-part".  this shoudl still compile but we shouldn't shorten full DB table names.

vishnuvellankijkt committed:
Ticket: #54690 Inbound API - CreateTagForPO

Files:
api\inbound\CreateTagForPO.p
api\inbound\handlers\CreateTagForPORequestHandler.p
